osu.Game.Tests/Visual/TestCaseMods.cs

```diff
@@ -182,13 +182,13 @@ private void testMultiplierTextColour(Mod mod, Color4 colour)
             checkLabelColor(Color4.White);
         }
 
-        private void testMultiplierTextUnranked(Mod mod)
+        private void testRankedText(Mod mod)
         {
-            AddAssert("check for ranked", () => !modSelect.MultiplierLabel.Text.EndsWith(unranked_suffix));
+            AddAssert("check for ranked", () => !modSelect.RankedLabel.Text.Equals(unranked_suffix));

```

Comment:
How do you check for fade?

Reply:
Which "fade" do you mean? Colour or Alpha? Either way you can check the property